project-docs/wave-03.md

```diff
@@ -1 +1,23 @@
-# Wave 03 - Like
\ No newline at end of file
+# Wave 03: Event Handling and Lifting Up State
+
+In this wave we will update the components to manage a **like** feature. 
+
+- Add behavior to heart button in `ChatEntry` so that when it is clicked it toggles from an empty heart (🤍) to a filled heart (❤️) and from a filled heart (❤️) to an empty heart (🤍).
```

Comment:
The heart icon is not already added to the component, right? It might be more clear if we said:

```suggestion
- Add a heart button and behavior in `ChatEntry` so that when it is clicked it toggles from an empty heart (🤍) to a filled heart (❤️) and from a filled heart (❤️) to an empty heart (🤍).
```

Reply:
The heart button is already added to the ChatEntry component, it jhust doesn't have any behavior